facebook/delphiFacebook/R/responses.R

```diff
@@ -511,11 +539,13 @@ module_assignment <- function(input_data, wave) {
 #'
 #' @param input_data data frame of responses
 #' @param county_crosswalk crosswalk mapping ZIP5 to counties
+#' @param params list containing `produce_individual_raceeth`, indicating
+#'   whether or not to issue microdata with race-ethnicity field
 #' @importFrom stringi stri_trim stri_replace_all
 #' @importFrom dplyr left_join group_by filter ungroup select rename
 #'
 #' @export
-create_complete_responses <- function(input_data, county_crosswalk)
+create_complete_responses <- function(input_data, county_crosswalk, params)
```

Comment:
...how did this work before?

Reply:
Something something environments. This does work in actual runs -- `params` is in the global environment and thus accessible to `create_complete_responses` -- but not in the integration tests, since `params` isn't defined globally there (the params are passed to `run_facebook` as an unnamed object).